src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -20,6 +20,16 @@ object AgoraEntity {
       case HttpEntity.Empty ⇒ new AgoraEntity()
     }
 
+  implicit val SeqAgoraEntityUnmarshaller =
+    Unmarshaller[Seq[AgoraEntity]](`application/json`) {
+      case HttpEntity.NonEmpty(contentType, data) => {
+        grater[AgoraEntity].fromJSONArray(data.asString)
+      }
+      case HttpEntity.Empty => {
+        Seq(new AgoraEntity())
```

Comment:
I think findSingle can still return an Option[AgoraEntity] it is just that the query method would have to convert that in to a singleton Seq[AgoraEntity] before marshalling it.

I'm all for less marshallers/unmarshallers and keeping our model simple, but I'm open to any suggestions.